build.sbt

```diff
@@ -1,29 +1,14 @@
 import Dependencies._
 
-ThisBuild / scalaVersion     := "2.13.4"
+ThisBuild / scalaVersion     := "2.12.13"
 ThisBuild / version          := "0.1.0-SNAPSHOT"
 ThisBuild / organization     := "com.example"
```

Comment:
`com.virtuslab` + `akka-safer-serializer` as module name to make it publish-ready